python/pyspark/sql/functions.py

```diff
@@ -1675,15 +1675,18 @@ def array(*cols):
 @since(1.5)
 def array_contains(col, value):
     """
-    Collection function: returns True if the array contains the given value. The collection
-    elements and value must be of the same type.
+    Collection function: returns null if the array is null, true if the array contains the
+    given value, and false otherwise.
```

Comment:
Other documentation use `true` rather than `True`. So, I matach this to `true`. I am willing to sweep if anyone feels this should be fixed.
The reason I removed `The collection elements and value must be of the same type` is it seems we can provide other types that are implicitly castable.
This is not documented in Scala/R too. So, I instead provided a doctest as an excpetion in the Python documentation.